src/ResourceManager/Common/Commands.ResourceManager.Common/Commands.ResourceManager.Common.csproj

```diff
@@ -68,6 +68,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.OperationalInsights, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Why is this added here?  This adds an OperationInsights dependency to all AzureRM projects - meaning evry time it's updated, every package would have to be republished.  Please add this only to your project